File: Ng.Net.Web/ClientApp/src/app/example/UserInfo.ts
/*
 * @Author: CollapseNav
 * @Date: 2020-01-03 20:05:15
 * @LastEditors  : CollapseNav
 * @LastEditTime : 2020-01-08 23:54:26
 * @Description:
 */
export class UserInfo {
  constructor(public id: string,
    public userAccount: string,
    public userName: string,
    public gender: string,
    public password: string,
    public createDate: Date,
    public phoneNumber: string, ) { }

}

File: Ng.Net.Repository/Domain/UserInfo.cs
using System.ComponentModel.DataAnnotations;
using Ng.Net.Repository.Core;

namespace Ng.Net.Repository.Domain
{
    public class UserInfo : BaseEntity
    {
        [Required, MaxLength(40)]
        public string UserAccount { get; set; }

        [MaxLength(18)]
        public string UserName { get; set; }

        [Required, MaxLength(30)]
        public string PassWord { get; set; }

        [MaxLength(11)]
        public string PhoneNumber { get; set; }

        [MaxLength(1)]
        public string Gender { get; set; }

    }
}
